src/ripple/overlay/impl/PeerImp.cpp

```diff
@@ -803,6 +817,36 @@ PeerImp::doProtocolStart()
 {
     onReadMessage(error_code(), 0);
 
+    // Send all the validator lists that have been loaded
+    if (supportsProtocolVersion("XRPL/2.1") )
```

Comment:
I don't quite like this; it requires manual parsing of a string for no real reason. It could simply boil down to:

```if (protocol_ >= make_protocol(2,1))```

Reply:
Agreed. Not sure why I thought this was a good idea...